snowflake-uar-reviews/README.md

```diff
@@ -0,0 +1,70 @@
+
+
+
+## What does the User Access Review app do?
+Provides an interface to re-evaluate user roles, access privileges, and credentials to mitigate insider threats, prevent insider mistakes, reduce excessive access, and meet compliance requirements such as HIPAA, FedRAMP, SOX, and more. Take actions such as approving or revoking privileges and enter your reasons for doing so. Submit individual changes or update multiple records in bulk. 
+
+This repo includes all code needed to source user roles within your Snowflake instance and generate the table schemas that the application attaches to. Once you load this application, you have the ability to modify the source code as you need to align to your own organization’s processes. This repo: 
+
+* Provides a schema to use on Snowflake
+* Provides a manager review application that implements filtering at the manager level
+* An admin application to get started with user and role entry
+* A review application to audit manager submissions for super-users and compliance teams
+
+## Caveats
+* If you need row-level security reads and writes, talk to your Retool account team
+* The template assumes a simple hierarchy (reviewer to employee), however you can customize this by modifying the schema
+* These applications are meant to be configurable to meet your needs 
+## Screenshots
+Manager Review Portal
+<p align="center">
+<img src="./images/manager_review_landing.png" alt="Manager Review Portal" style="width: 600px;">
+</p>
+Compliance Review Portal
+<p align="center">
+<img src="./images/compliance_review.png" alt="Manager Review Portal" style="width: 600px;">
+</p>
+Create Review Cycle Admin Application
+<p align="center">
+<img src="./images/create_cycle_name_cycle.png" alt="Manager Review Portal" style="width: 600px;">
+</p>
+<p align="center">
+<img src="./images/create_cycle_employees.png" alt="Manager Review Portal" style="width: 600px;">
+</p>
+<p align="center">
+<img src="./images/create_cycle_system_roles.png" alt="Manager Review Portal" style="width: 600px;">
+</p>
+
+## Technical Details + Setup
+1. Source the 3 applications from the retool/app-exchange repository: 
+    * Manager_Review_Portal.json
+    * Compliance_Review_Portal.json
+    * Create_Review_Cycle.json
+
+2. Import the app "code" in Retool
+On your Retool home page, click "Create new" → "Create a new folder". Give this folder a name you like, e.g. "Snowflake User Access Reviews". Now, let's import the app. Click "Create new" → "Import an app". Upload the application .json files from step 1. 
+
+
+
+3. Create the 4 tables used across the applications in your database: 
+    * _ACCESS_REVIEWS_
+    * _EMPLOYEES_
+    * _SYSTEM_ROLES_
+    * _REVIEW_CYCLES_
+4. Identify the employees and system roles for which we want to review:  
+    * The Create_Review_Cycle Retool application will allow you to manually input all the _EMPLOYEE_ and _SYSTEM_ROLES_ data required to generate the _ACCESS_REVIEWS_ table
```

Comment:
I'd remove underscores from app names for readability, unless you are referring to filenames directly.